eng/mgmt/automation/api-specs.yaml

```diff
@@ -47,6 +47,8 @@ network:
   suffix: generated
 operationalinsights:
   service: loganalytics
+postgresql:
+  service: postgresqlflexibleserver
```

Comment:
We might need to add some tag regex match here, so that e.g. `package-flexibleserver-XXX` goes to `postgresqlflexibleserver`, while other goes to `postgresql`.